src/app/pages/cookies/cookies-page.module.ts

```diff
@@ -0,0 +1,28 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { TranslateModule } from '@ngx-translate/core';
+
+import { CookiesModalComponent } from './cookies-modal/cookies-modal.component';
+import { CookiesPageGuard } from './cookies-page.guard';
+
+const cookiesPageRoutes: Routes = [
+  {
+    path: '',
+    children: [],
+    canActivate: [CookiesPageGuard],
+    data: {
+      meta: {
+        title: 'cookie.preferences.heading',
+        robots: 'noindex, nofollow',
+      },
+    },
+  },
```

Comment:
I don't think no bot is gonna go there. No meta required.

Reply:
![image](https://user-images.githubusercontent.com/50741106/96469744-d9182a80-122d-11eb-854d-e6f26d488c6b.png)
Isn't the link in the footer something a robot would follow and index if not told otherwise?

Reply:
Yes, you're right. But this part doesn't prevent the bot from going there (robots.txt does that), but the crawling won't continue after this page, so I guess we need this part, too.